src/functions-and-arrays.js

```diff
@@ -1,19 +1,65 @@
 // Iteration #1: Find the maximum
-
+const maxOfTwoNumbers = (number1, number2) => {
+  if (number1 > number2) {
+    return number1
+  } else {
+    return number2;
+  }
+}
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
+function findLongestWord(words) {
+  let longestWord = '';
+  if (words.length == '') {
+    return null
+  } else {
+    for (i = 0; i < words.length; i++) {
+      currentLength = words[i].length
+      if (currentLength > longestWord.length) {
+        longestWord = words[i];
+      }
+    }
+    return longestWord
+  }
+}
 
 // Iteration #3: Calculate the sum
 
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
+function sumNumbers(arr) {
+  let sum = 0;
+  for (let i = 0; i < arr.length; i++) {
+    sum += arr[i];
+  }
+  return sum;
+}
+sumNumbers(numbers);
 
 // Iteration #4: Calculate the average
 // Level 1: Array of numbers
 const numbersAvg = [2, 6, 9, 10, 7, 4, 1, 9];
 
+function averageNumbers(arr) {
+  if (arr.length === 0) {
+    return null;
+  }
+  return sumNumbers(arr) / arr.lengt;
```

Comment:
You're missing a "h" character at the end of line 46, in the future always read proof your code or run it before you submit.

Reply:
When I ran my code I didn't get any errors. Therefore, I wasn't aware that I forgot to add "h". Any suggestions for the plugin?
By the way, I will share my code with you again after I fix my code.